### PR TITLE
Finished developing e2e tests for /careers page

### PR DESCRIPTION
**Pull request by mjaskot**
Added corner case to checkCSSparams function and fixed css tests

### CODE REVIEW

File: cypress/integration/careers/careers_spec.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+      cy.get('[data-cy="photoRows"] > div:nth-child(1)')
+        .children()
+        .eq(1)
+        .should('exist')
+        .and('be.visible')
+      cy.get('[data-cy="photoRows"] > div:nth-child(1)')
+        .children()
+        .eq(2)
+        .should('exist')
+        .and('be.visible')
```
Comment (hypeofpipe) [inline cypress/integration/careers/careers_spec.js]: You can try to change this whole block to this:

```
Array(0, 1, 2).forEach(element =>
  cy
    .get('[data-cy="photoRows"]')
    .children()
    .each(child => cy.wrap(child).children().eq(element).should('exist').and('be.visible'))
)
```
This would shorten your code fragment from 30 lines to 6!

Anyway it would be shorter and more flexible (imagine, you have more children)

By the way, can't you don't check if the element exists, because you already check if the element is visible (if it visible => it should exist)?

File: cypress/integration/careers/careers_spec.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        .eq(1)
+        .should('exist')
+        .and('be.visible')
+      cy.get('[data-cy="photoRows"] > div:nth-child(2)')
+        .children()
+        .eq(2)
+        .should('exist')
+        .and('be.visible')
+
+      cy.get('[data-cy="photoCols"] > div:nth-child(1)')
```
Comment (hypeofpipe) [inline cypress/integration/careers/careers_spec.js]: Same thing, as in the previous comment

File: cypress/integration/careers/careers_spec.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        cy.log(`Checking "${element}" rendering`)
+        cy.contains(element)
+          .should('exist')
+          .and('be.visible')
+      })
+    })
+
+    it('Checks headers rendering', () => {
+      paragraphs.headers.forEach(element => {
+        if (element == 'Careers') {
```
Comment (hypeofpipe) [inline cypress/integration/careers/careers_spec.js]: Change it to 
``` 
if (element === 'Careers')
```
